client/app/components/proposed-development-types-list.js

```diff
@@ -0,0 +1,47 @@
+import Component from '@glimmer/component';
+
+export default class ProposedDevelopmentTypesListComponent extends Component {
+  get proposedDevelopmentTypes() {
+    const { pasForm } = this.args;
+
+    const developmentTypes = [
+      {
+        prop: 'dcpProposeddevelopmentsitenewconstruction',
+        label: 'Newly constructed buildings',
+      },
+      {
+        prop: 'dcpProposeddevelopmentsitedemolition',
+        label: 'Demolition',
+      },
+      {
+        prop: 'dcpProposeddevelopmentsiteinfoalteration',
+        label: 'Alteration',
+      },
+      {
+        prop: 'dcpProposeddevelopmentsiteinfoaddition',
+        label: 'Addition',
+      },
+      {
+        prop: 'dcpProposeddevelopmentsitechnageofuse',
+        label: 'Change of use',
+      },
+      {
+        prop: 'dcpProposeddevelopmentsiteenlargement',
+        label: 'Enlargement',
+      },
+      {
+        prop: 'dcpProposeddevelopmentsiteinfoother',
+        label: `Other (${pasForm.dcpProposeddevelopmentsiteotherexplanation})`,
+      },
+    ];
+
+    const arr = developmentTypes.reduce((proposedTypes, type) => {
+      if (pasForm[type.prop]) {
+        proposedTypes.push(type.label);
+      }
+      return proposedTypes;
+    }, []);
+
+    return arr.join(', ');
```

Comment:
Style thing but do you think this is easier to read than just returning `arr`?:

```javascript
    return developmentTypes
      .reduce((proposedTypes, type) => {
          if (pasForm[type.prop]) {
            proposedTypes.push(type.label);
          }
          return proposedTypes;
        }, [])
      .join(', ');
```

Reply:
You're right, I broke it out for debugging but should revert it.